spec/features/file_hierarchy_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'file viewer with hierarchy', js: true do
+  include PurlFixtures
+  it 'renders hierarchy' do
+    stub_purl_response_with_fixture(hierarchical_file_purl)
+    visit_iframe_response
+    expect(page).to have_content('2 items')
+    # There are 2 files
+    expect(page).to have_css('.sul-embed-media', count: 2)
+    # There are 2 directories
+    expect(page).to have_css('.sul-embed-treeitem', count: 2)
+    # One of the files is nested
+    expect(page).to have_css('[role="treeitem"] [role="treeitem"] [role="treeitem"] .sul-embed-media', count: 1)
+  end
+
+  it 'toggles hierarchy' do
+    stub_purl_response_with_fixture(hierarchical_file_purl)
+    visit_iframe_response
+
+    expect(page).to have_css('.sul-embed-media', count: 2)
```

Comment:
```suggestion
```

Can we combine these?  It's a single feature.  The feature tests are slow so we avoid having more than necessary.